topo/topo.go

```diff
@@ -629,9 +643,13 @@ func (s *sMap) TopoState() cpb.TopologyState {
 
 // GetTopologyServices returns the topology information.
 func GetTopologyServices(ctx context.Context, params TopologyParams) (*cpb.ShowTopologyResponse, error) {
-	topopb, err := Load(params.TopoName)
-	if err != nil {
-		return nil, fmt.Errorf("failed to load %s: %+v", params.TopoName, err)
+	var topopb *tpb.Topology
+	var err error
+	if params.TopoName != "" {
+		topopb, err = Load(params.TopoName)
```

Comment:
So this doesn't refer to the name of the topo, but rather the topo file path?

Reply:
It is referring to file path. I can have another PR to avoid the confusion.